packages/controller/src/dbClient.ts

```diff
@@ -19,7 +19,7 @@ import { getSdk } from "./dbSDK";
 // Make this optional, so that running the controller locally during dev
 // doesn't require a connection to the db.
 const endpoint = process.env.GRAPHQL_ENDPOINT;
-const adminSecret = process.env.HASURA_GRAPHQL_ADMIN_SECRET;
+const adminSecret = process.env.HASURA_GRAPHQL_ADMIN_SECRET ?? "";
```

Comment:
If this is a required input parameter and if an empty string is not an allowed value I think we should terminate the process here with a proper error message, saying that the environment variable `HASURA_GRAPHQL_ADMIN_SECRET` must be set to a non-empty value.

@MatApple what do you think?

Reply:
Actually we do terminate with an Error on the very next line if `adminSecret` is falsy (emtpy string is falsy) so this is fine. Good eye though.